routes/dbapi.js

```diff
@@ -110,15 +110,15 @@ db.prototype.recordGameStats = function(userId, payload, callback) {
 db.prototype.insertWordInCollection = function(payload, callback) {
   payload.word = payload.word.toLowerCase();
   _db.collection(gamesCollection).insert(payload, (err, result) => {
-    if (err) return callback(this.handleInsertionError(err));
-    return callback('Saved');
+    if (err) return callback(err);
+    callback(null, 'Saved');
```

Comment:
this code might/will break this 
https://github.com/prateekbh/Guess/blob/d7545feb7592b7312bbdc2ae6b6b54d78d5aa76d/routes/adminapi.js#L77